ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -67,6 +100,36 @@ QObject {
         return null
     }
 
+    function getSuggestedRoutes() {
+        let account = getSelectedAccount(root.swapFormData.selectedAccountIndex)
+        let accountAddress = account.address
+        let disabledChainIds = getDisabledChainIds(root.swapFormData.selectedNetworkChainId)
+        let preferedChainIds = getAllChainIds()
+
+        root.swapStore.suggestedRoutes(accountAddress, accountAddress, 
+            root.swapFormData.fromTokenAmount, root.swapFormData.fromTokensKey, root.swapFormData.toTokenKey,
```

Comment:
we should pass the actual required amount to be sent in this case. as this is not the right value to be sent in order to calculate routes.

Reply:
can we also rename this method in store to getSuggestedRoutes, also possible on nim side as it is more readable that way

Reply:
`AmountToSend` takes care of the conversion between the entered value (string representation of a real number) and the corresponding "wei" value (string representation of the u256 amount, considering the `decimals` value for the selected asset). Not sure how to include that here, but this is getting replaced soon so I guess it doesn't matter that much?

Reply:
yes but its not being used as of now here, so best to adda TODO to remove it under that task but add it here, so that things still work